client/components/input/InputText.vue

```diff
@@ -18,5 +22,40 @@ defineExpose({
 </script>
 
 <template>
-  <PInputText v-bind="$attrs" ref="el" v-model="value" :class="{ 'p-invalid': errorMessage }" />
+  <label :for="id" class="block font-900 font-medium mb-2">{{ label }}</label>
+  <div class="mb-3">
+    <span class="w-full mb-1" :class="{ 'p-input-icon-left': startIcon, 'p-input-icon-right': endIcon }">
+      <i v-if="startIcon" :class="startIcon" />
+      <PInputText :id="id" v-bind="$attrs" ref="el" v-model="value" :class="{ 'p-invalid': errorMessage }" />
+      <i v-if="endIcon" :class="endIcon" />
+    </span>
+    <Transition>
+      <small v-if="errorMessage" id="text-error" class="block p-error">{{ errorMessage }}</small>
```

Comment:
id has to be unique on a single page, per html semantics. If you use this component several times on the same page, then several elements with the id text-error will be loaded.

Make the id depend on the prop id

Reply:
Yup forgot this..